microbial_thermodynamics/cell_growth_ODE.py

```diff
@@ -112,6 +112,108 @@ def da(
     return uda
 
 
+# metabolic functions
+
+
+def calculate_kappa(
+    Gatp: float = 75,
+    G0: float = 1.5e5,
+    reaction_energy: float = 1 / 3,
+    R: float = 8.314,
+    T: float = 293.15,
+) -> float:
+    """Calculates the kappa constant."""
+    u = np.exp((-G0 - reaction_energy * Gatp) / R * T)
+    return u
+
+
+def calculate_theta(s: float, w: float) -> float:
+    """A thermodynamic factor that stops a reaction at equilibrium.
+
+    Args:
+     s: substrate concentration
+     w: waste product concentration
+     kappa: species equilibrium constant.
+    """
+    kappa = calculate_kappa()
```

Comment:
`calculate_kappa` should be called with the value for `reaction_energy` provided